qiskit/transpiler/passes/routing/cython/stochastic_swap/utils.pxd

```diff
@@ -31,7 +31,8 @@ cdef class NLayout:
 
 
 cpdef NLayout nlayout_from_layout(object layout,
-                                  object dag, 
+                                  object dag,
```

Comment:
Is there a reason this has a different name than the implementation in `utils.pyx`? I probably would make this also typed as a `dict`. So something like:

```suggestion
                                  dict qubit_indices,
```

Reply:
Agree, added in 23a6597 .